demoextendtemplates/views/PrestaShop/Admin/Sell/Order/Order/index.html.twig

```diff
@@ -0,0 +1,42 @@
+{% extends '@PrestaShopCore/Admin/Sell/Order/Order/index.html.twig' %}
+
+{# modify the content block #}
+{% block content %}
+  {# call parent if you want to see all the original content rendered#}
+  {{ parent() }}
+  <div class="text-success">
+    <p>Extended block content in demoextendtemplates module</p>
+  </div>
+{% endblock %}
+
+
+{% block order_grid_row %}
+  {#
+    Modify a block which is not directly rendered in extended template, but in some included file.
+    We need to "embed" the same file that is included in original index.html.twig
+    and then we can modify the blocks inside it.
+    Note that we also need to pass the variables as the orderGrid in this example.
```

Comment:
Maybe it would be good to mention that we also have a hook available to use before and after each `Grid` table?